### DISCUSSION TITLE
skywalking-client-js Can we extend support for custom properties in perfData and errorLogs?

### DISCUSSION THREAD

**Discussion by 13992764153** [apache/skywalking#12404] [Q&A]
Question: On the same page, I reported a perfData and segments (with custom attributes in tags) at 13:00, and a perfData and segments (without custom attributes in tags) at 13:01. I want to associate the perfData and segments at 13:00 with a unique identifier. Also, at 13:01, I reported a perfData and segments that are associated. Is there a solution? Can perfData and errlog be extended with a custom attribute field, similar to tags in span

**Comment by wu-sheng** [comment]
No, you can't and should not think in this way. perfData are metrics, which will be aggregated to metrics you see on the UI.

**Comment by 13992764153** [comment]
okay, thank you
